python/lsst/analysis/tools/actions/scalar/scalarActions.py

```diff
@@ -432,5 +435,12 @@ def __call__(self, data: KeyedData, **kwargs) -> Scalar:
         scalarA = self.actionA(data, **kwargs)
         scalarB = self.actionB(data, **kwargs)
         if scalarB == 0:
-            raise ValueError("Denominator is zero!")
-        return scalarA / scalarB
+            if scalarA == 0:
+                log.warning("Both numerator and denominator are zero! Returning NaN.")
+                return np.nan
+            else:
+                value = np.sign(scalarA) * np.inf
```

Comment:
You can ignore me since jenkins is running and you want to merge but doesn't this work:

```suggestion
                value = np.copysign(np.inf, scalarA)
```
?

I also have a general comment. Why are we using numpy here at all? There are no vectors being manipulated and it's all explicitly scalar. What's the objection to doing:

```python
value = math.copysign(math.inf, scalarA)
```
?

Reply:
Jenkins passed. I'm merging this one, but I'll consider your suggestion on [another ticket](https://rubinobs.atlassian.net/browse/DM-45306) that I just opened to add tests for this error handling.